test/renate/updaters/test_model_updater.py

```diff
@@ -204,6 +207,23 @@ def identity_transform():
     check_learner_transforms(model_updater._learner, transforms_kwargs)
     model = model_updater.update(train_dataset, task_id=defaults.TASK_ID)
     model_updater = pytest.helpers.get_simple_updater(
-        model, input_state_folder=state_url, learner_class=learner_class, **transforms_kwargs
+        model,
+        input_state_folder=state_url,
+        learner_class=learner_class,
+        **transforms_kwargs,
+        learner_kwargs=LEARNER_KWARGS[learner_class],
     )
     check_learner_transforms(model_updater._learner, transforms_kwargs)
+
+
+@pytest.mark.xfail(raises=(KeyError, TypeError))
+@pytest.mark.parametrize("learner", LEARNERS_USING_SIMPLE_UPDATER)
+def test_learner_fails_without_loss_fn(learner):
```

Comment:
I don't think this test is particular useful. maybe you want to check if they use the loss instead?